installer/bundle/bundle_skel.sh

```diff
@@ -871,10 +911,8 @@ case "$installMode" in
 
             pkg_add_list $OMS_PKG omsagent
 
-            python_ctypes_installed
-            if [ $? -eq 0 ]; then
-                pkg_add_list $DSC_PKG omsconfig
-            fi
+            shouldInstall_omsconfig
```

Comment:
The update path uses the result of this line in `pkg_upd_list $DSC_PKG omsconfig $?`, but the `pkg_add_list` subroutine doesn't have a parameter like that. I believe we have to keep the check if the return type is 0 before we call `pkg_add_list`:

```
shouldInstall_omsconfig
if [ $? -eq 0 ]; then
    pkg_add_list $DSC_PKG omsconfig
fi
```